Source/WebKit/UIProcess/API/ios/WKWebViewIOS.mm

```diff
@@ -2296,7 +2284,14 @@ - (void)_frameOrBoundsMayHaveChanged
             _page->setDefaultUnobscuredSize(WebCore::FloatSize(bounds.size));
         [self _recalculateViewportSizesWithMinimumViewportInset:_minimumViewportInset maximumViewportInset:_maximumViewportInset throwOnInvalidInput:NO];
 
-        [self _updateDrawingAreaSize];
```

Comment:
I wonder if this change can be left alone, since it's a correctness fix and at least partly fixes the original bug?